test/codegen/newman/fixtures/basicCollection.json

```diff
@@ -280,6 +280,32 @@
 			},
 			"response": []
 		},
+		{
```

Comment:
We should also add a unit test for the codegens that do not run newman tests.
@shreys7

Reply:
on it

Reply:
added unit tests @umeshp7